option.go

```diff
@@ -60,12 +73,12 @@ func TimeLayout(layout string) Option {
 }
 
 // UseEnv returns an option that configures fig to additionally load values
-// from the environment, after it has loaded values from a config file.
+// from the environment.
 //
 //   fig.Load(&cfg, fig.UseEnv("my_app"))
 //
-// This is meant to be used in conjunction with loading from a file. There
-// is no support to ONLY load from the environment.
+// By default config values are first loaded from the config file and then from
+// the environment. But if `NoFile` option is also given then config file is omitted.
```

Comment:
We could simplify this by instead saying something along the lines of "Values loaded from the environment overwrite values loaded by the config file (if any)."